### PR TITLE
DOC-7557 Added new infrastructure alert conditions procedure

### PR DESCRIPTION
**Pull request by paperclypse**
I added a new procedure to describe how to create an alert condition in the Infrastructure settings UI.

### CODE REVIEW

**Comment by paperclypse** [comment]
This one is still a work-in-progress. Not ready for PR review.

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx [hunk 1]
```diff
@@ -54,52 +54,49 @@ To add an infrastructure alert condition to an alerts policy:
...
+    <Collapser
+    className="freq-link"
+    id="infrastructure-ui"
+    title="Use the Infrastructure UI"
+  >
+    Go to **[one.newrelic.com](http://one.newrelic.com) > Infrastructure**, then select any of these Infrastructure monitoring pages: [**Hosts**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-compute-page-measure-resource-usage), [**Processes**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-processes-page-inspect-process-performance), [**Network**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-network-page-measure-compare-capacity), or [**Storage**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-storage-page-evaluate-disk-usage-efficiency). Mouse over the chart you want to alert on, select the ellipses <Icon name="fe-more-horizontal"/> icon, and then select **Create alert**.
+  </Collapser>
+    <Collapser
+    className="freq-link"
+    id="infrastructure-settings"
```
Comment (barbnewrelic) [inline src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:82]: I wonder if you should call this id "integration" rather than "infrastructure-settings" because the original "integration" ID (line 57) has been deleted?

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx [hunk 2]
```diff
@@ -54,52 +54,49 @@ To add an infrastructure alert condition to an alerts policy:
...
-  <tbody>
-    <tr>
-      <td>
-        From **Alerts & AI**
-      </td>
-
-      <td>
-        Go to **[one.newrelic.com](http://one.newrelic.com) > Alerts & AI** **> Alerts >** **Alert policies > New alert policy > Create new condition**, then select **Infrastructure** as the product.
+<Callout variant="important">
+  If New Relic hasn't received a Cloud Integration service's attribute in the past 60 minutes, then it won't be selectable for alerting in the UI. You can use the [API](/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/rest-api-calls-new-relic-infrastructure-alerts/) to create an alert condition for silent attributes.
```
Comment (barbnewrelic) [inline src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:58]: I don't think Cloud Integration needs to be initial caps.
Maybe reword "then it won't be selectable for alerting in the UI" to "then you can't select it in the UI" or "then it won't be an alerting option in the UI"?
Last sentence: I think I know what "silent attributes" means, but it caught me by surprise. Maybe reword to something like:
If New Relic hasn't received a cloud integration service's attribute in the past 60 minutes, we refer to this as a "silent attribute," and it won't be available to use as an alert condition in the UI. In this situation, you can use the API to create alert conditions for silent attributes.

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx [hunk 3]
```diff
@@ -54,52 +54,49 @@ To add an infrastructure alert condition to an alerts policy:
...
+  >
+    Go to **[one.newrelic.com](http://one.newrelic.com) > Alerts & AI** **> Alerts >** **Alert policies > New alert policy > Create new condition**, then select **Infrastructure** as the product.
+  </Collapser>
+    <Collapser
+    className="freq-link"
+    id="infrastructure-ui"
+    title="Use the Infrastructure UI"
+  >
+    Go to **[one.newrelic.com](http://one.newrelic.com) > Infrastructure**, then select any of these Infrastructure monitoring pages: [**Hosts**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-compute-page-measure-resource-usage), [**Processes**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-processes-page-inspect-process-performance), [**Network**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-network-page-measure-compare-capacity), or [**Storage**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-storage-page-evaluate-disk-usage-efficiency). Mouse over the chart you want to alert on, select the ellipses <Icon name="fe-more-horizontal"/> icon, and then select **Create alert**.
```
Comment (barbnewrelic) [inline src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:78]: Maybe consider breaking this into individual steps rather than a single long paragraph for easier skimming?

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx [hunk 4]
```diff
@@ -54,52 +54,49 @@ To add an infrastructure alert condition to an alerts policy:
...
+
+You can also use these other methods to create an infrastructure alert condition:
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="alerts-ui"
+    title="Use the Alerts UI"
+  >
+    Go to **[one.newrelic.com](http://one.newrelic.com) > Alerts & AI** **> Alerts >** **Alert policies > New alert policy > Create new condition**, then select **Infrastructure** as the product.
```
Comment (barbnewrelic) [inline src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:71]: I don't have a test account to look at policy setup in UI but am assuming "product" is still the right word/ field name?

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx [hunk 5]
```diff
@@ -54,52 +54,49 @@ To add an infrastructure alert condition to an alerts policy:
...
+    className="freq-link"
+    id="infrastructure-ui"
+    title="Use the Infrastructure UI"
+  >
+    Go to **[one.newrelic.com](http://one.newrelic.com) > Infrastructure**, then select any of these Infrastructure monitoring pages: [**Hosts**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-compute-page-measure-resource-usage), [**Processes**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-processes-page-inspect-process-performance), [**Network**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-network-page-measure-compare-capacity), or [**Storage**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-storage-page-evaluate-disk-usage-efficiency). Mouse over the chart you want to alert on, select the ellipses <Icon name="fe-more-horizontal"/> icon, and then select **Create alert**.
+  </Collapser>
+    <Collapser
+    className="freq-link"
+    id="infrastructure-settings"
+    title="Use the Infrastructure settings"
```
Comment (barbnewrelic) [inline src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:83]: Maybe change title to "Use infrastructure settings for integrations" so the collapser is clear what problem this solves?

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx [hunk 6]
```diff
@@ -54,52 +54,49 @@ To add an infrastructure alert condition to an alerts policy:
...
+</Callout>
 
-        Go to **[one.newrelic.com](http://one.newrelic.com) > Alerts & AI** **> Alerts >** **Alert policies > (select a policy)**, then select **Add a condition**.
-      </td>
-    </tr>
+1. Go to **[one.newrelic.com](http://one.newrelic.com)** > **Infrastructure** > **Settings** > **Alerts**, and then click **Create alert condition**.
+2. Name and describe the alert condition.
+3. Click the **Integrations** alert type, and then select the integration data source you'd like to use.
+4. Use the **Filter entities** dropdown to limit your condition to specific attributes.
+5. Use the **Define thresholds** dropdowns to define your condition's thresholds and then click **Create**.
```
Comment (barbnewrelic) [inline src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:94]: thresholds, and then
^^ add comma like step 2 does

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx [hunk 7]
```diff
@@ -54,52 +54,49 @@ To add an infrastructure alert condition to an alerts policy:
...
-      </td>
-
-      <td>
-        Go to **[one.newrelic.com](http://one.newrelic.com) > Infrastructure**, then select any of these Infrastructure monitoring pages: [**Hosts**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-compute-page-measure-resource-usage), [**Processes**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-processes-page-inspect-process-performance), [**Network**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-network-page-measure-compare-capacity), or [**Storage**](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-storage-page-evaluate-disk-usage-efficiency). Mouse over the chart you want to alert on, select the ellipses <Icon name="fe-more-horizontal"/>
-        icon, and then select **Create alert**.
-      </td>
-    </tr>
-  </tbody>
-</table>
+The configuration settings are optional and you can always update them later.
```
Comment (barbnewrelic) [inline src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:96]: Seems like 2 separate thoughts. "optional. You" ?

**Review by barbnewrelic — CHANGES_REQUESTED** [review]
I like the flow of this, and it looks fine in my localhost. I've made a lot of minor copyedit suggestions, take 'em or leave 'em! Then feel free to merge your PR with your admin privileges. No need for second review from me. Nice job!

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx (hunk 2 above)
Comment (paperclypse) [reply src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:58]: Good call!

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx (hunk 4 above)
Comment (paperclypse) [reply src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:71]: Yup, it's still "product" in the UI

File: src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx (hunk 3 above)
Comment (paperclypse) [reply src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx:78]: I was considering this, but your opinion has tipped me over to doing it :)

**Comment by paperclypse** [comment]
Good suggestions! Thanks, @barbnewrelic!
